explorer/app/models/viewModels.ts

```diff
@@ -1,7 +1,10 @@
+import { DetailResponseType } from "./responses";
+
 //Project
-export interface DetailViewModel {
+export interface DetailModel {
   json?: string;
```

Comment:
What's the difference between `json` and `data`? Is `json` just the raw API response?

Reply:
Let's discuss this on our call today! It looks like either `json` or `data` could be specified, depending on the use of the `DetailModel`?

Reply:
As discussed, remove `json`.